docs/app/components/usage-link/usage-link.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Usage } from './../../interfaces/Usage';
+import { Input, Renderer2, Component } from '@angular/core';
+
+@Component({
+    selector: 'uxd-usage-link',
+    templateUrl: './usage-link.component.html',
+    styleUrls: ['./usage-link.component.less'],
+    host: {
+        '[class.enabled]': 'isUsageEnabled()'
```

Comment:
Is this needed anymore (and the function isUsageEnabled)? as we use an ngIf not to show this component at all if there is no usage value.

if the off chance that this is needed you should change it to:

```
'[class.enabled]': 'usage'
```

As calling a function is much less performant than just accessing a value as it can trigger change detection